scripts/linters/pylint_extensions.py

```diff
@@ -2109,6 +2109,46 @@ def visit_call(self, node):
                     break
 
 
+class DisallowBlankLinesBelowFunctionDefinitionChecker(checkers.BaseChecker):
+    """Custom pylint checker to ensure there are no blank lines
+    below function definition.
+    """
+
+    __implements__ = interfaces.IAstroidChecker
+    name = 'disallow-blank-lines-below-function-definition'
+    priority = -1
+    msgs = {
+        'C0034': (
+            'Please remove the blank line(s) below the function definition.',
+            'remove-blank-lines-below-function-definiton.',
```

Comment:
```suggestion
            'no-blank-lines-below-function-definiton',
```

The checker name should explain what practices it brings on the codebase. Note: we use this name as a pragma so it should be well defined.

Reply:
done. Changed 'remove' with no.